tests/test_bouquets/test_bouquets.py

```diff
@@ -0,0 +1,8 @@
+from tests.base import BaseTestCase
+
+
+class TestChannels(BaseTestCase):
+    def test_get_all_channels(self):
```

Comment:
You have to rename this class and test method

Reply:
Resolved